src/consumer/component/consumer-component.js

```diff
@@ -712,13 +712,24 @@ export default class Component {
 
       let specsResults: RawTestsResults[];
 
+      let contextPaths;
+      if (this.tester && this.tester.context) {
+        contextPaths = this.tester.context;
+      } else if (consumer && consumer.bitMap) {
+        contextPaths = {
+          componentDir: consumer.bitMap.projectRoot
```

Comment:
I'm not sure this is what you expect to get.
The project root is not the component dir but the workspace dir.
you probably want to use:
this.componentMap.getTrackDir